src/form/Calendar/Calendar.tsx

```diff
@@ -208,6 +209,7 @@ export const Calendar: FC<CalendarProps> = ({
           className={css.leftArrow}
           disablePadding
           onClick={previousClickHandler}
+          style={{ color: 'var(--white)' }}
```

Comment:
??

Reply:
According to the new Figma design, made this change.

Reply:
as austin mentioned, we don't allow inline styles